akka-actor/src/main/scala/akka/actor/ActorRef.scala

```diff
@@ -48,7 +49,7 @@ object ActorRef {
  *     case Request2(msg) => other.tell(msg, sender()) // forward sender reference, enabling direct reply
  *     case Request3(msg) =>
  *       implicit val timeout = Timeout(5.seconds)
- *       sender() ! (other ? msg)  // will reply with a Future for holding other's reply
+ *       (other ? msg) pipeTo sender() // will get a Future from other and pipe its value to sender
```

Comment:
```from other``` I don;t think so.
the ```ask``` return a ```future``` of the other's reply.and the future is comes from the local ask support actor not the other that is asked.
how about 
```the ask call will get a Future of other's first reply,and then we pipe its value to the sender when it complete.```